config/app.php

```diff
@@ -81,6 +81,12 @@
     */
 
     'locale' => 'hu',
+    'locales' => [
+    	'en' => 'English',
+    	'hu' => 'Magyar',
+    	'it' => 'Italiano',
+    ],
```

Comment:
Please add latin ('lat') since it's also merged.

Reply:
In my opinion we should stick with [ISO-639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) codes, and use _la_ for Latin, furthermore I think the name should be _Latina_ (or something like that) since that's used on the [Latin Wikipedia](https://la.wikipedia.org/wiki/Lingua_Latina).